database/migrations/2022_10_06_154300_create_contacts_table.php

```diff
@@ -12,12 +12,14 @@ public function up(): void
         Schema::create('contacts', static function (Blueprint $table) {
             $table->uuid('id')->primary();
             $table->uuid('integration_id');
+            $table->string('email');
             $table->string('type');
             $table->string('first_name');
             $table->string('last_name');
-            $table->string('email');
             $table->softDeletes();
             $table->timestamps();
+
+            $table->unique(['integration_id','email']);
```

Comment:
I think some organizations might have the same person for both the technical & the functional contact.
That would mean $table->unique(['integration_id','type','email']);